superset/config.py

```diff
@@ -279,10 +279,11 @@ def _try_json_readsha(filepath: str, length: int) -> str | None:
 
 # Add endpoints that need to be exempt from CSRF protection
 WTF_CSRF_EXEMPT_LIST = [
-    "superset.views.core.log",
-    "superset.views.core.explore_json",
     "superset.charts.data.api.data",
     "superset.dashboards.api.cache_dashboard_screenshot",
+    "superset.views.core.explore_json",
+    "superset.views.core.log",
+    "superset.views.datasource.views.samples",
```

Comment:
Just really adding `"superset.views.datasource.views.samples". Moved the other two to keep the list alphabetically sorted